Cargo.toml

```diff
@@ -163,8 +163,6 @@ codegen-units = 1
 unexpected_cfgs = { level = "warn", check-cfg = ['cfg(zcash_unstable, values("zfuture"))'] }
 
 [patch.crates-io]
-sapling-crypto = { git = "https://github.com/zcash/sapling-crypto", rev = "b1ad3694ee13a2fc5d291ad04721a6252da0993c" }
-orchard = { git = "https://github.com/zcash/orchard", rev = "55fb089a335bbbc1cda186c706bc037073df8eb7" }
 incrementalmerkletree = { git = "https://github.com/zcash/incrementalmerkletree", rev = "ffe4234788fd22662b937ba7c6ea01535fcc1293" }
 incrementalmerkletree-testing = { git = "https://github.com/zcash/incrementalmerkletree", rev = "ffe4234788fd22662b937ba7c6ea01535fcc1293" }
```

Comment:
Can you drop the `incrementalmerkletree` patch dependency as well? You'll still need `incrementalmerkletree-testing`.

Reply:
No, as I said in our pairing earlier this breaks while `shardtree` is a patch dependency.

Reply:
Because you then have two versions of `incrementalmerkletree` in the dependency tree; you still get the git version via `shardtree`, but now it isn't unified with `sapling-crypto` and `orchard`, and you get trait incoherence errors.

Reply:
https://github.com/zcash/incrementalmerkletree/pull/116 fixes this issue.